tsl/test/expected/continuous_aggs-12.out

```diff
@@ -1555,9 +1555,8 @@ insert into raw_data select '2000-05-01 00:00+0','Q3', 0, 0;
 --this one fails now
 \set ON_ERROR_STOP 0
 CALL refresh_continuous_aggregate('search_query_count_3', NULL, '2000-06-01 00:00+0'::timestamptz);
-ERROR:  cannot update/delete rows from chunk "_hyper_41_79_chunk" as it is compressed
 CALL refresh_continuous_aggregate('search_query_count_3', '2000-05-01 00:00+0'::timestamptz, '2000-06-01 00:00+0'::timestamptz);
-ERROR:  cannot update/delete rows from chunk "_hyper_41_79_chunk" as it is compressed
+NOTICE:  continuous aggregate "search_query_count_3" is already up-to-date
 \set ON_ERROR_STOP 1
```

Comment:
It works now, so let's update the comment and remove ON_ERROR_STOP.

Reply:
Fixed.